katharsis-core/src/main/java/io/katharsis/legacy/queryParams/params/FilterParams.java

```diff
@@ -16,6 +18,21 @@ public FilterParams(Map<String, Set<String>> params) {
     }
 
     @Override
+    public int hashCode() {
```

Comment:
Please use the standard hash/equals generators from IJ - I would preferr wholesale move to Lombok